bot.py

```diff
@@ -332,6 +361,36 @@ async def listen_for_response(
 discord.utils.setup_logging(handler=handler)
 
 
+@bot.command(
+    name=SYNC_COMMAND_NAME,
+    description="Syncs command list. Any new commands will be available for use.",
+)
+@commands.check(is_staff_from_ctx)
+async def sync(ctx: commands.Context, only_guild_commands: bool = False):
```

Comment:
My only concern is that the boolean argument has an ambiguous meaning when typing it out in Discord (since there doesn't seem to be a way to display help info for a legacy command?).

I feel it would be better instead for this argument to be a string and to to check for `"all" | "guild" | "guildonly" | "guild-only"` and prompt an error msg stating command usage if the user puts something different.